pages/index.js

```diff
@@ -24,35 +26,44 @@ let ingredients = [
 ];
 
 export default function Home() {
+  const [ingredients, setIngredients] = useState(defaultIngredients);
+
   function editBagle(ingredient) {
-    if (ingredient.chosen) {
-      ingredient.chosen = false;
-    } else {
-      ingredient.chosen = true;
-    }
+    ingredient.chosen = !ingredient.chosen;
   }
 
   return (
     <div>
+      <Head>
+        <title>Parkbench Bagels</title>
+      </Head>
+
       <h1>Parkbench Bagels</h1>
 
       <h2>Build your own Bagle!</h2>
+      <form
+        onSubmit={(e) => {
+          // only for Testing
+          e.preventDefault();
+          console.log(e.t);
+        }}
```

Comment:
You wrote that this part is only for testing. Why did you include it in your Pull Request if you don't need it as it is now for the functionality of your app? 🤔 And I would recommend to use meaningful names that are understandable for other developer and also write `event` instead of `e`. :)